src/calculateRentalCost.js

```diff
@@ -27,8 +27,16 @@
  * @return {number}
  */
 
-function calculateRentalCost(days) {
-  // write code here
+function calculateRentalCost(numberOfDays) {
+  const DAY_COST = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const MIDDLE_TERM = 3;
+  const MIDDLE_TERM_DISCOUNT = 20;
+  const total = DAY_COST * numberOfDays;
+
+  return numberOfDays >= LONG_TERM ? total - LONG_TERM_DISCOUNT
+    : numberOfDays >= MIDDLE_TERM ? total - MIDDLE_TERM_DISCOUNT : total;
```

Comment:
its smart but I think it would be way easier to read if you'd use `if else` statement

Reply:
done in 'Fixed soultion' ; )

Reply:
You could rewrite it to avoid `else if` according to checklist point 4. Then it will be perfect